docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -279,6 +204,12 @@ spec:
             name: worker-user-data
 ```
 
```

Comment:
Shoud we mention the the creation of the NodePolicy here from? maybe just link the example from dpdk?

Reply:
Good idea.

Reply:
done